content/release-10-18+/announcements-10-18+.md

```diff
@@ -139,6 +139,21 @@ This change affects the following APIs:
 * `DELETE /measurements/measurement` requires at least one of the following parameters: `source`, `dateFrom`, `dateTo`
 
 
+##### Change in full text search feature of Inventory API
+
+Starting with 2024 release only following properties are included into full text search functionality:
```

Comment:
@c8y-kh4mell Due to the CD model, we no longer talk about releases here but about a time frame. Does this change stick to the 6-month period for announcements? I assume it will if you talk about 2024 release?

Reply:
Generally this change will arrive within next GA after 10.18